resources/META-INF/plugin.xml

```diff
@@ -21,6 +21,17 @@
 
   <change-notes>
     <![CDATA[
+
+0.12.0:
```

Comment:
Yes, I suggested we make this 1.0 (as in our first, complete and stable plugin). It seems little odd to me to start with 12.0.